templates/pages/admin/form/form_destination_commonitil_config.html.twig

```diff
@@ -43,24 +43,34 @@
         {% if field.supportAutoConfiguration() %}
             {% set use_auto_configuration = field.isAutoConfigurated(config) %}
             {% set auto_configuration_checkbox %}
-                <label class="form-check form-switch ms-auto mb-0">
-                    <input
-                        name="{{ item.formatConfigInputName(field.getAutoConfigKey()) }}"
-                        type="hidden"
-                        value="0"
-                        data-glpi-itildestination-toggle-do-not-disable
-                    >
-                    <input
-                        name="{{ item.formatConfigInputName(field.getAutoConfigKey()) }}"
-                        class="form-check-input"
-                        type="checkbox"
-                        value="1"
-                        {{ use_auto_configuration ? 'checked' : '' }}
-                        data-glpi-itildestination-toggle-auto-config
-                        data-glpi-itildestination-toggle-do-not-disable
-                    >
-                    <span class="form-check-label">{{ __("Auto config") }}</span>
-                </label>
+                <div class="ms-auto row g-2 me-2">
+                    <label class="col form-check form-switch mb-0">
+                        <input
+                            name="{{ item.formatConfigInputName(field.getAutoConfigKey()) }}"
+                            type="hidden"
+                            value="0"
+                            data-glpi-itildestination-toggle-do-not-disable
+                        >
+                        <input
+                            name="{{ item.formatConfigInputName(field.getAutoConfigKey()) }}"
+                            class="form-check-input"
+                            type="checkbox"
+                            value="1"
+                            {{ use_auto_configuration ? 'checked' : '' }}
+                            data-glpi-itildestination-toggle-auto-config
+                            data-glpi-itildestination-toggle-do-not-disable
+                        >
+                        <span class="form-check-label">{{ __("Auto config") }}</span>
+                    </label>
+                    <span class="col-auto form-help align-self-center"
+                        data-bs-toggle="popover"
+                        data-bs-trigger="hover"
+                        data-bs-placement="top"
+                        data-bs-html="true"
+                        data-bs-content="{{ __('The auto configuration option allows dynamically configuring the content of the created object based on the different fields of the form. <br> <br> <strong>Warning</strong>: The data entered in the field will be overwritten by the auto configuration.')|raw }}">
```

Comment:
Using HTML tags in translations is prohibited. Indeed, translators should not have to know the HTML language to be able to translate the application, and there is a risk that a translator breaks the HTML code (and breaks the whole page also).
Either you have to split into multiple translations strings, either you have to remove some layout.

Reply:
I guess the warning could be removed.